roles/dnsmasq/templates/dnsmasq-ds.yml

```diff
@@ -25,7 +25,7 @@ spec:
             capabilities:
               add:
                 - NET_ADMIN
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
+1 specific tags are ok. Upgrade will change the tag and then the new image will download fine